rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -892,6 +892,60 @@ void testSaveOldFederation(
         );
     }
 
+    @Test
+    void savePendingFederation_postWasabi_shouldBeSavedInStorageWithFormatVersion() {
+
+        // Arrange
+
+        ActivationConfig.ForBlock wasabiActivation = ActivationConfigsForTest.wasabi100().forBlock(0L);
+        PendingFederation expectedPendingFederation = new PendingFederationBuilder().build();
+        StorageAccessor storageAccessor = new InMemoryStorage();
+
+        // Act
+
+        FederationStorageProvider federationStorageProvider = new FederationStorageProviderImpl(storageAccessor);
+        federationStorageProvider.setPendingFederation(expectedPendingFederation);
+        federationStorageProvider.save(networkParameters, wasabiActivation);
+
+        PendingFederation actualPendingFederationInStorage = storageAccessor.getFromRepository(PENDING_FEDERATION_KEY.getKey(),
+            PendingFederation::deserialize);
+
+        int formatVersion = storageAccessor.getFromRepository(PENDING_FEDERATION_FORMAT_VERSION.getKey(), BridgeSerializationUtils::deserializeInteger);
+
+        // Assert
+
+        assertEquals(STANDARD_MULTISIG_FEDERATION.getFormatVersion(), formatVersion);
+        assertEquals(expectedPendingFederation, actualPendingFederationInStorage);
+
+    }
+
+    @Test
+    void savePendingFederation_preWasabi_shouldBeSavedInStorageSerializedFromBtcKeysOnly() {
```

Comment:
```suggestion
    void savePendingFederation_preWasabi_shouldBeSavedInStorageSerializedFromBtcKeysOnly() {
```

This test should be before the postWasabi

Reply:
Already discussed in the poll.